### ISSUE TITLE
Precompiled .apk File Missing

### ISSUE THREAD

**Issue by MoneyMikePMZ** [microsoft/appcenter-Xamarin.UITest-Demo#9]
The master branch is missing the precompiled .apk file from the source. As a workaround, I pulled it from another branch. According to the [README.md](https://github.com/microsoft/appcenter-Xamarin.UITest-Demo/blob/master/README.md) the is required to test locally using a windows machine

> "You must use the .ApkFile() method to point to the APK."

Can the apk file be added back to the master branch?

**Comment by King-of-Spades** [comment]
@moneymikepmz 
Yeah, there's pros & cons to having a precomplied APK. The advantage is, you can test Xamarin.UITest without building an APK at all, but the disadvantage is I have to manually maintain that APK including updating it & checking it regularly. What finally broke it is the fact that the APK blocks integration with App Center Build signing.  

I did forget to update the README so I'll try to get that done. You can generate an APK using the Xamarin.Android project to build it.

**Comment by MRoy-BTE** [comment]
@King-of-Spades thanks you for the quick reply. I am completely new to mobile development so I was using this project to learn about UI Testing. I am not sure of how to generate an APK using the  Xamarin.Android project so the README update will be very useful.

**Comment by King-of-Spades** [comment]
@MRoy-BTE you might want to check out https://docs.microsoft.com/en-us/xamarin/android/get-started/

There's also a Xamarin.Android FAQ here: https://docs.microsoft.com/en-us/xamarin/android/troubleshooting/questions/

I helped write a lot of that content back when I was on that team, actually. But the publish date is back in summer of 2018, so some of it might be outdated, unfortunately.
